### ISSUE TITLE
Json2XmlValidator returns faulty xml when validating json message that doesn't adhere to XSD

### ISSUE THREAD

**Issue by BonnoSmeele** [ibissource/iaf#2896]
**❗ please do not add sensitive information in issues, you can provide extra information via email using issue number as reference ❗**

**Describe the issue**
I'm using an Json2XmlValidator pipe to convert json to xml and test it against xsd scheme. When json is passed that is technically correct but does violates the XSD the validator detects the error as expected but does not pass the completed xml translation.
It looks like the end(root) tag is missing

**Reporter**
Bonno!

**To Reproduce**
1. create a Json2XmlValidator
2. use json with an unexpected element(i use 'aaa' in the example)
3. pipe output
4. xml is missing end tag

**Screenshots**
![image](https://user-images.githubusercontent.com/73215587/159235845-db0c6d9b-8584-4b57-8189-a5c1d1487111.png)
![image](https://user-images.githubusercontent.com/73215587/159235912-a8daa32c-0882-4e68-8152-39d249dc04e6.png)

**Configuration**
			<Json2XmlInputValidator
				name="validator"
				schemaLocation="urn:items /HandleBP/xsd/HandleBP.xsd" 
				root="BusinessPartner" 
				responseRoot="Result"
				storeResultInSessionKey="originalMessage">
				<Forward name="failure" 	  path="createLogMessageValidationError"/>
				<Forward name="outputFailure" path="createLogMessageValidationError"/>
			</Json2XmlInputValidator>

**Input**
```
Please provide an example of the input message. 
Alternatively, provide a Ladybug report.
```
[Pipeline HandleBPAdapter.zip](https://github.com/ibissource/iaf/files/8314380/Pipeline.HandleBPAdapter.zip)

**Environment**
FF! 7.7-20220202.144505: ibis4salesforce_notify
Running on LPZWNL000013838 using Apache Tomcat/9.0.56
Java Version: OpenJDK Runtime Environment (1.8.0_292-b10)
Heap size: 380M, total JVM memory: 487M
Free disk space: 12GB, total disk space: 237GB
Up since: 2022-03-20 20:02:43 (14h)

**Additional Environment**
 - DBMS: [e.g. Oracle, MSSQL, MariaDB, MySql, PostgreSQL]
 - Browser: [e.g. Chrome, Safari, Edge, Firefox]

**Additional Context**
Add any other context about the problem here. (f.e. ladybug report / test adapter with larva test)

**Comment by gvanbrakel** [comment]
you could set the attribute 'ignoreUndeclaredElements' to true, to ignore the element.
Or do you want the validation to fail, but still have well-formed XML?

**Comment by BonnoSmeele** [comment]
Hi Gerrit,

Unofrtunately i need to get values for logging from the payload(for splunk), so that's not an option.

**Comment by gvanbrakel** [comment]
> Hi Gerrit,
> 
> Unofrtunately i need to get values for logging from the payload(for splunk), so that's not an option.

This is addressed in PR #2918
